internal/useragent/useragent.go

```diff
@@ -0,0 +1,15 @@
+package useragent
```

Comment:
You need to add the Copyright and file description at the beginning of the document.